reference/build_helpers/cmake.rst

```diff
@@ -145,6 +145,10 @@ The CMake helper will automatically append some definitions based on your settin
 +-------------------------------------------+--------------------------------------------------------------------------+
 | CONAN_LINK_RUNTIME                        |  Runtime from self.settings.compiler.runtime for MSVS                    |
 +-------------------------------------------+--------------------------------------------------------------------------+
+| CONAN_CMAKE_CXX_STANDARD                  |  From setting ``cppstd``                                                 |
```

Comment:
Missing CONAN_STANDARD_FLAG?

Reply:
? There is no CONAN_STANDARD_FLAG. Do you want me to rename it?

Reply:
Sorry ``CONAN_STD_CXX_FLAG``